Dockerfile

```
@@ -1,13 +1,12 @@
-FROM node:21-alpine
+FROM imbios/bun-node:latest-21-slim
```

```suggestion
FROM oven/bun:1
```

Now you are running it in actual bun (vite etc) you shouldn't need node here anymore